packages/core-data/src/test/selectors.js

```diff
@@ -25,11 +24,7 @@ import {
 	getRevision,
 } from '../selectors';
 
-// getEntityRecord and __experimentalGetEntityRecordNoResolver selectors share the same tests.
-describe.each( [
-	[ getEntityRecord ],
-	[ __experimentalGetEntityRecordNoResolver ],
-] )( '%p', ( selector ) => {
```

Comment:
I've removed coverage for `__experimentalGetEntityRecordNoResolver`. It's a leftover from when the data package was introduced and hasn't been used in the core.

I think we should just remove or deprecate it.